clients/unshaded/src/main/java/tachyon/client/RemoteBlockInStream.java

```diff
@@ -163,6 +173,13 @@ public void close() throws IOException {
       mCheckpointInputStream.close();
     }
     mClosed = true;
+    if (mBytesReadRemote > 0) {
+      mTachyonFS.incBytesReadRemote(mBytesReadRemote);
+      mTachyonFS.incBlocksReadRemote(1);
+    }
+    if (mBytesReadUfs > 0) {
+      mTachyonFS.incBytesReadUfs(mBytesReadUfs);
```

Comment:
Do we need to wait until the Block is closed before incrementing the read bytes?